src/learn/concepts/accounts.md

```diff
@@ -1 +1,48 @@
-# Accounts
+---
+Author: [Cedric](syyanyuhui@gmail.com)
+LastUpdate: 2024-05-06 
+---
+
+## Address Types in Aleo
+
+In Aleo, there are two types of address: user address and program address.
+
+User address have unique corresponding ViewKey and PrivateKey. In the Aleo network, we can create programs with privacy protection, where interactions between users and programs are invisible to everyone else. The **privacy of user addresses** is provided by the ViewKey, meaning whoever possesses the ViewKey corresponding to a user address has visibility over that address. The PrivateKey serves as proof of **ownership of the user address**; whoever possesses the PrivateKey corresponding to a user address owns all assets associated with that address. Address, ViewKey, and PrivateKey form a closely related set of mathematical objects designed based on cryptography. PrivateKey can derive ViewKey, and ViewKey can derive Address. This process is one-way; we cannot reverse-engineer ViewKey from Address or derive PrivateKey from ViewKey.
```

Comment:
Some minor changes I'd recommend
- visibility over *the records belonging to* that address
- "The PrivateKey can derive the ViewKey, and the ViewKey can derive the Address"
- "this process is one-way; we cannot reverse-engineer the ViewKey from the Address or derive the PrivateKey from the ViewKey."